src/client/src/ui/blotter/epics/blotterServiceEpic.ts

```diff
@@ -60,4 +60,18 @@ const connectBlotterToNotifications: ApplicationEpic = (action$, state$, { platf
     ignoreElements(),
   )
 
-export const publishBlotterEpic = combineEpics(connectBlotterToExcel, connectBlotterToNotifications)
+export const requestBrowserNotificationPermission: ApplicationEpic = (action$, state$, { platform }) =>
+  action$.pipe(
+    applicationConnected,
+    delay(3000),
```

Comment:
Why a 3 second delay? This is waiting for some other async event to have taken place?

Reply:
3 second delay removed, it was not waiting for any async event, the notification permissions request now happens instantly